questions/206/explanation.md

```diff
@@ -11,10 +11,10 @@ The grammar for `sizeof` is in §[expr.unary]¶1:
 
 We have three cases and the one that applies here is **sizeof unary-expression**. The *unary expression* is `(0)["abcdefghij"]`, which looks odd but is just array indexing of *string literal* which is a *const char array*. 
 
-We can see that `(0)["abcdefghij"]` is identical to `("abcdefghij")[0]` from §[expr.sub]¶1 which says:
+We can see that `(0)["abcdefghij"]` is identical to `("abcdefghij")[0]` from §[expr.sub]¶2 which says:
 
 > ... The expression `E1[E2]` is identical (by definition) to `*((E1)+(E2))` ...
 
 So we end up with 0th element of `"abcdefghij"`, which is `a`, which is a `char`. And the result of `sizeof('a')` will be `1` since §[expr.sizeof]¶1 says:
 
-> ... `sizeof(char)`, `sizeof(signed char)` and `sizeof(unsigned char)` are `1` ...
+> The result of sizeof applied to any of the narrow character types is 1.
```

Comment:
Maybe it's also worth explaining that `char` is a narrow character type like [here](https://github.com/knatten/cppquiz23/pull/218).

Reply:
Yeah I think that would be good.